runtime/queries/rust/highlights.scm

```diff
@@ -33,7 +33,9 @@
 
 ; Namespaces
 
-(crate) @namespace
+(extern_crate_declaration
+    (crate)
+    name: (identifier) @namespace)
```

Comment:
You should keep the old definition as well, or you break highlighting for 

```
crate::test;
```

Reply:
I reverted it.